core/frontend/src/menus.ts

```diff
@@ -0,0 +1,134 @@
+        route: '/autopilot/general',
+        advanced: false,
+        text: 'General Autopilot settings, allow you to start/stop Ardupilot if using Navigator or SITL, and switch'
+              + ' boards if more than one is present',
+      },
+      {
+        title: 'Firmware',
+        icon: 'mdi-chip',
+        route: '/autopilot/firmware',
+        advanced: false,
+        text: 'Used to download and flash new firmware to your board',
+      },
+      {
+        title: 'Log Browser',
+        icon: 'mdi-file-multiple',
+        route: '/autopilot/logs',
+        advanced: false,
+        text: 'Allow browsing the Telemetry (.tlog) and Binary (.bin) logs generated by your vehicle. Bin logs are'
+              + ' currently only supported for Navigator boards',
+      },
+      {
+        title: 'Endpoints',
+        icon: 'mdi-arrow-decision',
+        route: '/autopilot/endpoints',
+        advanced: true,
+        text: 'Manage Mavlink endpoints for internal/external systems. Used this if you need to connect additional'
+              + ' mavlink systems to your vehicle',
+      },
+      {
+        title: 'Video',
+        icon: 'mdi-video-vintage',
+        route: '/autopilot/video-manager',
+        advanced: false,
+        text: 'Manage your video devices and video streams',
+      },
+    ],
+  },
+  {
+    title: 'Tools',
+    icon: 'mdi-hammer-screwdriver',
+    submenus: [
+      {
+        title: 'Available Services',
+        icon: 'mdi-account-hard-hat',
+        route: '/tools/available-services',
+        advanced: true,
+        text: 'List all available services found in BlueOS serving http interfaces, and their'
+              + ' respective API documentations',
+      },
+      {
+        title: 'Bridges',
+        icon: 'mdi-bridge',
+        route: '/tools/bridges',
+        advanced: true,
+        text: 'Allows creating UDP/TCP to Serial bridges, used for communication to serial'
+              + ' devices from your topside computer',
+      },
+      {
+        title: 'File Browser',
+        icon: 'mdi-file-tree',
+        route: '/tools/file-browser',
+        advanced: true,
+        text: 'Browse all the files in BlueOS, useful for fetching logs,'
+              + ' tweaking configurations, and development',
+      },
+      {
+        title: 'NMEA Injector',
+        icon: 'mdi-map-marker',
+        route: '/tools/nmea-injector',
+        advanced: true,
+        text: 'Used for forwarding UDP NMEA stream into Ardupilot',
+      },
+      {
+        title: 'System information',
+        icon: 'mdi-chart-pie',
+        route: '/tools/system-information',
+        advanced: false,
+        text: 'Detailed system status information, CPU, memory, disk, and ethernet status',
+      },
+      {
+        title: 'Network test',
+        icon: 'mdi-speedometer',
+        route: '/tools/network-test',
+        show: true,
+        text: 'Test link speed between topside computer and your vehicle',
```

Comment:
I like this for sub, although I'm assuming drone users don't refer to the GCS as a "topside" computer? Not sure what our preference is here (do we want default sub terminology, or actively keep things general to ArduPilot vehicles?)

Reply:
we're currently walking the line... I don't want to be in any sides too much :sweat_smile: 
see my previous comment about "flight controller"

Reply:
I can just remove the "topside" reference